docs/releases/v1.16.0.md

```diff
@@ -0,0 +1,64 @@
+
+This minor release updates some components and adds support to Kubernetes 1.27.
+
+## Component Images 🚢
+
+| Component         | Supported Version                                                                | Previous Version |
+| ----------------- | -------------------------------------------------------------------------------- | ---------------- |
+| `calico`          | [`v3.27.0`](https://projectcalico.docs.tigera.io/archive/v3.26/release-notes/)   | `v3.26.1`        |
+| `cilium`          | [`v1.15.0`](https://github.com/cilium/cilium/releases/tag/v1.13.1)               | `v1.13.3`        |
+| `ip-masq`         | [`v2.8.0`](https://github.com/kubernetes-sigs/ip-masq-agent/releases/tag/v2.5.0) | `v2.8.0`       |
+| `tigera-operator` | [`v1.32.3`](https://github.com/tigera/operator/releases/tag/v1.30.4)             | `v1.30.4`        |
+
+> Please refer the individual release notes to get detailed information on each release.
+
+## Update Guide 🦮
+
+### Process
+
+If you are using Cilium, read the steps [below](#cilium-upgrade) before proceeding.
+
+1. Just deploy as usual:
+
+```bash
+kustomize build katalog/calico | kubectl apply -f -
+# OR
+kustomize build katalog/tigera/on-prem | kubectl apply -f -
+# OR
+kustomize build katalog/cilium | kubectl apply -f -
+```
+
+#### Cilium upgrade
+Cilium suggested path expect a pre-flight check to be run before any upgrade.
+
+1. Create the resources for the check
+```bash
+kubectl create -f katalog/cilium/tasks/preflight.yaml
+```
+
+2. Ensure that the number of READY pods is the same number of Cilium pods running.
+```text
+kubectl get daemonset -n kube-system | sed -n '1p;/cilium/p'
+NAME                      DESIRED   CURRENT   READY   UP-TO-DATE   AVAILABLE   NODE SELECTOR   AGE
+cilium                    2         2         2       2            2           <none>          1h20m
+cilium-pre-flight-check   2         2         2       2            2           <none>          7m15s
+```
+
+3. Once the number of READY pods is equal, make sure the Cilium pre-flight deployment is also marked as READY 1/1.
+If it shows READY 0/1, consult the [CNP Validation](https://docs.cilium.io/en/stable/operations/upgrade/#cnp-validation) section in the official docs and resolve issues with the deployment before continuing with the upgrade.
+```text
+kubectl get deployment -n kube-system cilium-pre-flight-check -w
+NAME                      READY   UP-TO-DATE   AVAILABLE   AGE
+cilium-pre-flight-check   1/1     1            0           12s
+```
+
+4. Once the number of READY for the preflight DaemonSet is the same as the number of cilium pods running and the preflight Deployment is marked as READY 1/1 you can delete the cilium-preflight and proceed with the upgrade.
+```bash
+kubectl delete -f cilium-preflight.yaml
+```
+
+
+If you are upgrading from previous versions, please refer to the [`v1.15.0` release notes](https://github.com/sighupio/fury-kubernetes-networking/releases/tag/v1.14.0).
```

Comment:
The upgrade guide must be the same as the destination version.
```suggestion
If you are upgrading from previous versions, please refer to the [`v1.15.0` release notes](https://github.com/sighupio/fury-kubernetes-networking/releases/tag/v1.15.0).
```

Reply:
Fixed in https://github.com/sighupio/fury-kubernetes-networking/commit/054c8312bf9742049ee1acce0a7de14f2ab51c2b